spec/feature/login_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'Hello world', type: :feature do
```

Comment:
## [OPTIONAL]
- [ ] Kindly add meaningful descriptions to these tests. kindly Remove the `hello wold`.